GLRenderer/GLRenderer/Textures/Texture.cpp

```diff
@@ -20,21 +21,28 @@ namespace GLEngine::GLRenderer::Textures {
 
 //=================================================================================
 C_Texture::C_Texture(const std::string& name, GLenum target)
```

Comment:
This ctor should be removed

Reply:
Working on it. I need to fix the ray tracing window and it will be possible to delete this.